nuget/Plugin.nuspec

```diff
@@ -62,6 +62,10 @@
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.dll" target="lib\wpa81\Plugin.Media.Abstractions.dll" />
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.xml" target="lib\wpa81\Plugin.Media.Abstractions.xml" />
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.pdb" target="lib\wpa81\Plugin.Media.Abstractions.pdb" />
+     <file src="src\Media.Plugin.WindowsPhone81\bin\Release\CameraCaptureUI.xaml.cs" target="lib\wpa81\CameraCaptureUI.xaml.cs" />
```

Comment:
Hi, the problem is that if you don't add XAML files gives a XAML Parsing Exception. I have tested beta5 and the problem continues. I added those lines because finally I used as a library but it would be perfect if could be added to nu get output.

Thank you very much